### PR TITLE
add change mngmt for enterprise article

### PR DESCRIPTION
**Pull request by nataliejeremy**
This closes issue #408 @bmackinney please review and add additional info if needed.

### CODE REVIEW

File: source/docs/articles/sites/enterprise/changemanagement/change-management-for-enterprise.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Change Management for Enterprise Organizations
+description: All about Drupal on Pantheon.
```
Comment (bmackinney) [inline source/docs/articles/sites/enterprise/changemanagement/change-management-for-enterprise.md]: ?

File: source/docs/articles/sites/enterprise/changemanagement/change-management-for-enterprise.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: Change Management for Enterprise Organizations
+description: Learn about the features and benefits of role-based permissions for Pantheon Enterprise Organizations.
+keywords: roles, role, permissions, permission, access, privileges, privilege, team members, developer, organization, enterprise, manage organization, change management, enterprise, one
+---
+Change Management is a feature of Organizations that enables role-based permissions for users in the Organization. Sites which list the Organization as a supporting Organization are accessible to privileged members of the Organization. The roles exist to restrict who can deploy and manage other users in the Organization or sites it works on.
```
Comment (ari-gold) [inline source/docs/articles/sites/enterprise/changemanagement/change-management-for-enterprise.md]: "supporting Organization" >> "Supporting Organization" ?

**Comment by rachelwhitton** [comment]
The breadcrumbs include two landing pages that do not currently exist: 
screen shot 2015-06-16 at 10 11 56 am

I suggest moving the enterprise doc to the Organizations dir or creating the landing pages and cleaning up the file path so it renders properly in the breadcrumbs.

File: source/docs/articles/sites/enterprise/changemanagement/change-management-for-enterprise.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+5. Click the **Add Agency** button to make them a Supporting Organization.
+
+## Site Roles and Permissions
+From your Org Dashboard, click **Sites**, select a site, then click **Team**. You will see the site owner, team members and their role, and any Supporting Organizations. 
+
+### Team Member
+Team Members are a trusted role that can make changes to the Test and Live environments, add and edit other team members, and add Supporting Organizations.
+
+### Developer
+Developers can only work on the Development environment, and cannot deploy to Test or Live.
```
Comment (rachelwhitton) [inline source/docs/articles/sites/enterprise/changemanagement/change-management-for-enterprise.md]: Should we include that they can affect code on CDEs/Multidev Environments?

Comment (nataliejeremy) [inline source/docs/articles/sites/enterprise/changemanagement/change-management-for-enterprise.md]: @rachelwhitton Please feel free to add anything you think is useful for Enterprise customers! This is just a starting point and I'm hoping for input.

**Comment by nataliejeremy** [comment]
in  PR #650

**Pull request closed without merging**